app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split(" ")
+    mv = command[0]
+    file_to_move = command[1]
+    destination = command[2].split("/")
+    if mv == "mv" and file_to_move != "".join(destination):
+        try:
+            with open(file_to_move, "r") as f:
+                data = f.read()
+            os.remove(file_to_move)
+        except FileNotFoundError:
+            print("File not found")
+        path = ""
+        if len(destination) == 1:
+            path = destination[0]
+        else:
+            for i in range(len(destination) - 1):
+                path += destination[i] + "/"
+                if not os.path.exists(path):
+                    os.mkdir(path)
```

Comment:
Code Style issue: You are using string concatenation to create the path. This is not a good practice because it's not cross-platform. You should use 'os.path.join' to concatenate parts of the path. In that case, your app will be cross-platform and will use either '/' or '\' depending on the OS.